splice_machine/src/main/java/com/splicemachine/derby/impl/sql/compile/HalfMergeSortJoinStrategy.java

```diff
@@ -144,29 +144,39 @@ public int maxCapacity(int userSpecifiedCapacity, int maxMemoryPerTable, double
     private boolean mergeable(IndexRowGenerator innerRowGenerator,
```

Comment:
This class is not used, it can be removed

Reply:
Done.